src/cluster.h

```diff
@@ -288,9 +286,19 @@ typedef struct {
                                             master is up. */
 
 /* ---------------------- API exported outside cluster.c -------------------- */
+void clusterInit(void);
+void clusterCron(void);
+void clusterBeforeSleep(void);
 clusterNode *getNodeByQuery(client *c, struct redisCommand *cmd, robj **argv, int argc, int *hashslot, int *ask);
+clusterNode *clusterLookupNode(const char *name);
 int clusterRedirectBlockedClientIfNeeded(client *c);
 void clusterRedirectClient(client *c, clusterNode *n, int hashslot, int error_code);
+void migrateCloseTimedoutSockets(void);
+int verifyClusterConfigWithData(void);
 unsigned long getClusterConnectionsCount(void);
+int clusterSendModuleMessageToTarget(const char *target, uint64_t module_id, uint8_t type, unsigned char *payload, uint32_t len);
+void clusterPropagatePublish(robj *channel, robj *message);
+unsigned short crc16(const char *buf, int len);
```

Comment:
Do we need this? I would imagine we should just import crc16.h in cluster.c.

Reply:
Wanted to do that... but unfortunately there's no crc16.h :(
I would be happy to create it, but it would prob. be better to do that as part of cleaning up & modularizing the crc* files.

Reply:
I would vote to remove the server.h declaration from crc16.c and add a crc16.h now, unless you think strongly we'll refactor this more aggressively into a single crc.h header.

Reply:
I'd prefer to avoid adding crc16.h here -- seems a bit out of scope for this PR (esp. since I think a subdir + common header for all hash function related stuff is the best approach for crc*+sha*).

In order to avoid creating merge conflicts multiple times, maybe I should avoid touching it in this PR (leaving it in server.h) and only address it when we do the above? does that sound OK to you?

Reply:
Sounds good to me. Agree, would be better to just move it once.

Reply:
done.